test/runtests.cmd

```diff
@@ -174,6 +174,11 @@ goto :main
   if /i "%1" == "-variants"         set _Variants=%~2&                                          goto :ArgOkShift2
   if /i "%1" == "-cleanupall"       set _CleanUpAll=1&                                          goto :ArgOk
 
+  ::Extra ch.exe command line flags
+  if /i "%1" == "-ExtraCCFlags"     set _ExtraCCFlags=%~2&                                      goto :ArgOkShift2

```

Comment:
Can we call this `ExtraHostFlags` instead to move away from the C legacy, at least in the public facing interface?